app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Dwarf(Player, ABC):
```

Comment:
No need to inherit from ABC. Class that doesn't implement all abstract methods will remain abstract